layouts/section/partnerships.html

```diff
@@ -8,55 +8,55 @@
         <p>{{ i18n "partnerships-intro" | safeHTML }}</p>
       </div>
     </div>
-    <!-- How it works -->
-    <div class="row">
-      <div class="col-xs-12">
-        <h2>{{ i18n "partnerships-how-it-works" | safeHTML }}</h2>
-        <p>{{ i18n "partnerships-how-we-do-it-1st" | safeHTML }}</p>
-        <p>{{ i18n "partnerships-how-we-do-it-2nd" | safeHTML }}</p>
-        <p>{{ i18n "partnerships-how-we-do-it-3d" | safeHTML }}</p>
-        <p>{{ i18n "partnerships-how-we-do-it-4th" | safeHTML }}</p>
-      </div>
-    </div>
   </div>
 </section>
-
-<section class='bg-dark-full'>
-  <!-- Who is on the team -->
-  <div class='container'>
-    <div class='row'>
-      <div class='col-xs-12'>
-        <h2>{{ i18n "partnerships-team" | safeHTML }}</h2>
-        <p>{{ i18n "partnerships-team-blurb" | safeHTML }}</p>
-
-        <p>{{ i18n "partnerships-cds" | safeHTML }}</p>
+<section class="partnerships-info bg-med-grey">
+  <div class="container">
+    <div class="row">
+      <div class="col-xs-12">
+        <h2 class="partnership-title">{{i18n "partnerships-what-we-can-help-title" | safeHTML}}</h2>
```

Comment:
I didn't see any HTML in the various values, why do these all have `| safeHTML` on them?

Reply:
Sorry, think it was lingering from before and never removed it. fixed now.